lib/src/widgets/detailCardsWidgets/custom_appbar.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:flutter/material.dart';
+import '../../utils/ui_constants.dart';
+
+class CustomAppBar extends StatelessWidget {
+  const CustomAppBar({
+    Key? key,
+    required this.posterPath,
```

Comment:
Instead of naming this a posterPath lets just name it image right ? so when you use it outside posters it make sense

Reply:
i agree , renamed.